### PR TITLE
[enterprise-4.17+]: Update network-flow-matrix ports

### PR DESCRIPTION
**Pull request by sabinaaledort**
Add missing ports and fix wrong info.

Version(s):
4.17

Link to docs preview:
https://80339--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall#network-flow-matrix_configuring-firewall

### CODE REVIEW

File: snippets/network-flow-matrix.csv [hunk 1]
```diff
@@ -1,14 +1,16 @@
 Direction,Protocol,Port,Namespace,Service,Pod,Container,Node Role,Optional
 Ingress,TCP,22,Host system service,sshd,,,master,TRUE
 Ingress,TCP,53,openshift-dns,dns-default,dnf-default,dns,master,FALSE
+Ingress,TCP,80,openshift-ingress,router-default,router-default,router,master,FALSE
 Ingress,TCP,111,Host system service,rpcbind,,,master,TRUE
+Ingress,TCP,443,openshift-ingress,router-default,router-default,router,master,FALSE
```
Comment (EricPonvelle) [inline snippets/network-flow-matrix.csv:6]: Looks fine here:
image

File: snippets/network-flow-matrix.csv [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 Ingress,TCP,2379,openshift-etcd,etcd,etcd,etcdctl,master,FALSE
 Ingress,TCP,2380,openshift-etcd,healthz,etcd,etcd,master,FALSE
 Ingress,TCP,5050,openshift-machine-api,,ironic-proxy,ironic-proxy,master,FALSE
 Ingress,TCP,6080,openshift-kube-apiserver,,kube-apiserver,kube-apiserver-insecure-readyz,master,FALSE
 Ingress,TCP,6385,openshift-machine-api,,ironic-proxy,ironic-proxy,master,FALSE
 Ingress,TCP,6443,openshift-kube-apiserver,apiserver,kube-apiserver,kube-apiserver,master,FALSE
-Ingress,TCP,8080,openshift-network-operator   ,,network-operator,network-operator,master,FALSE
+Ingress,TCP,8080,openshift-network-operator,,network-operator,network-operator,master,FALSE
```
Comment (EricPonvelle) [inline snippets/network-flow-matrix.csv:13]: Looks good:
image

File: snippets/network-flow-matrix.csv [hunk 3]
```diff
@@ -60,7 +62,7 @@ Ingress,TCP,9637,openshift-machine-config-operator,kube-rbac-proxy-crio,kube-rba
 Ingress,TCP,10250,Host system service,kubelet,,,worker,FALSE
 Ingress,TCP,10256,openshift-ovn-kubernetes,ovnkube,ovnkube,ovnkube-controller,worker,TRUE
 Ingress,TCP,10300,openshift-cluster-csi-drivers,csi-livenessprobe,csi-driver-node,csi-driver,worker,FALSE
-Ingress,TCP,10309,openshift-cluster-csi-drivers,csi-node-driver-registrar,csi-driver-node,csi-node-driver-registrar,worker,FALSE
+Ingress,TCP,10309,openshift-cluster-csi-drivers,csi-node-driver,csi-driver-node,csi-node-driver-registrar,worker,FALSE
```
Comment (EricPonvelle) [inline snippets/network-flow-matrix.csv:65]: Looks fine here too:
image

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Comment by ShaunaDiaz** [comment]
@sabinaaledort I see you opened another CP post-merge, but you did not indicate 4.16 in your opening commit message. Do you have a Jira issue for this work?

**Pull request merged**